resources/qml/delegates/TextMessage.qml

```diff
@@ -29,6 +29,11 @@ MatrixText {
         border-collapse: collapse;
         border: 1px solid " + Nheko.colors.text + ";
     }
+    blockquote {
+        margin: 0;
+        margin-left: 1em;
+    }
+    blockquote p { margin: 0; }
```

Comment:
This seems redundant?

Reply:
`<blockquote>` and `<p>` both have margins.

Reply:
Without `blockquote p { margin: 0; }`: 
![screenshot_2021-09-04T17:38:50](https://user-images.githubusercontent.com/3681516/132100146-88e2334d-0d32-4bbf-b850-47d5c9ab0f79.png)

Without `blockquote { margin: 0; }`: 
![screenshot_2021-09-04T17:40:18](https://user-images.githubusercontent.com/3681516/132100147-8cf731ee-cf6c-49e3-9f53-cb54ac369997.png)